gulpfile.js

```diff
@@ -1,6 +1,7 @@
 require('source-map-support').install();
 
 var formatter = require('gulp-clang-format');
+var fs = require('fs');
```

Comment:
fs-extra extends fs, so you could just use fsx below
but, since that is surprising, I'm also okay with being explicit here